README.adoc

```diff
@@ -331,6 +331,60 @@ spec:
 ...
 ----
 
+== Services API Documentation
+
+Kiali can display API Documentation of your services. See https://user-images.githubusercontent.com/1235410/61569413-0e870480-aa3c-11e9-9527-bbea18eff475.png[Api documentation screen capture] and https://user-images.githubusercontent.com/1235410/61569419-1777d600-aa3c-11e9-82cc-f9510e5b9808.png[Api type list screen capture].
+
+=== Configure your services
+
+Your services must be annotated with the type of api ('rest', 'grpc', 'graphql') and a url to the spec of the api. 
```

Comment:
minor, inconsistent capitalization. Should probably use "API" consistently. Also, URL.

Reply:
Fixed